terracumber-cli

```diff
@@ -44,6 +44,8 @@ def parse_args():
                                           new module were added). Valid when using --runall or
                                           --runstep provision""",
                         action='store_true')
+    parser.add_argument('--terraformbin', help="""path to terraform bin that should be used""",
```

Comment:
No need for tripple quotes here, as this is a single line. Also, should we maybe call the parameter `--terraform-bin` to make it easier to read? You did it for the variable.

Reply:
I will remove tripple quotes.
Regarding the name I follow the same logic as the other parameters and have all concatenated. 
If you think it should be change in this direction I'm fine with it

Reply:
or others we don't use a separator, but somehow I find `terraformbin` hard to identify, that's also why we have `--custom-repositories` instead of `--customrepositories`. `gitref` or `gitfolder` are not hard to identify, or maybe it is just me